shpc/main/modules/__init__.py

```diff
@@ -219,17 +219,20 @@ def create_symlink(self, module_dir):
         # Create .version
         self.write_version_file(os.path.dirname(symlink_path))
 
-    def check_symlink(self, module_dir):
+    def check_symlink(self, module_dir, force):
         """
         Given an install command, if --symlink-tree is provided make
         sure we don't already have this symlink in the tree.
         """
         # Get the symlink path - does it exist?
         symlink_path = self.get_symlink_path(module_dir)
-        if os.path.exists(symlink_path) and not utils.confirm_action(
-            "%s already exists, are you sure you want to overwrite?" % symlink_path
-        ):
-            return False
+        if os.path.exists(symlink_path):
+            if force:
+                logger.info("Overwriting %s, as requested")
```

Comment:
```suggestion
                logger.info(f"Overwriting {symlink_path}, as requested")
```

Reply:
Went for the traditional %-formatting because there are no f-strings in shpc so far. It felt odd introducing the first one here

Reply:
Sounds good! I've only started to start using format strings myself (for other new libraries) and i suspect at some point might do a sweep through old ones to change the style. It's funny I still prefer `%s` when the string is kind of long/gnarly or otherwise weird (e.g., index of a dictionary always is weird). Anyway +1 to this choice.